src/scopes/activityReport/feiRootCause.js

```diff
@@ -0,0 +1,21 @@
+import { filterAssociation } from './utils';
+
+const feiResponseSql = `
+WITH unnested_responses AS (
+    SELECT "activityReportGoalId", unnest("response") AS res
+    FROM "ActivityReportGoalFieldResponses"
+)
+SELECT
+  DISTINCT "ActivityReportGoals"."activityReportId"
+FROM "ActivityReportGoals" "ActivityReportGoals"
+INNER JOIN unnested_responses arr
+    ON arr."activityReportGoalId" = "ActivityReportGoals"."id"
+WHERE arr."res"`;
+
+export function withFeiRootCause(responses) {
```

Comment:
With the route written generically, why is the scope bespoke to only the root cause, it is was written to be generic then as others get added, it would already be filterable.

Reply:
This is only for FEI Root Cause on AR atm.